### PR TITLE
Pass lib ID to toolbar so it can refresh state when clearing issues.

### PR DESCRIPTION
**Pull request by barrycarey**
This seems like a clean way to do this.  Although I've never used Vuex so feel free to slap my wrist if this isn't right.

### CODE REVIEW

File: client/pages/library/_library/bookshelf/_id.vue [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <template>
   <div class="page" :class="streamLibraryItem ? 'streaming' : ''">
-    <app-book-shelf-toolbar :page="id || ''" :view-mode.sync="viewMode" />
+    <app-book-shelf-toolbar :page="id || ''" :view-mode.sync="viewMode" :library-id="libraryId"/>
```
Comment (advplyr) [inline client/pages/library/_library/bookshelf/_id.vue:3]:
```suggestion
    <app-book-shelf-toolbar :page="id || ''" :view-mode.sync="viewMode" />
```

File: client/components/app/BookShelfToolbar.vue [hunk 2]
```diff
@@ -89,7 +89,8 @@ export default {
     authors: {
       type: Array,
       default: () => []
-    }
+    },
+    libraryId: String
```
Comment (advplyr) [inline client/components/app/BookShelfToolbar.vue:93]:
```suggestion
    }
```

File: client/components/app/BookShelfToolbar.vue [hunk 3]
```diff
@@ -215,6 +216,7 @@ export default {
             this.$toast.success('Removed library items with issues')
             this.$router.push(`/library/${this.currentLibraryId}/bookshelf`)
             this.processingIssues = false
+            this.$store.dispatch('libraries/fetch', this.libraryId)
```
Comment (advplyr) [inline client/components/app/BookShelfToolbar.vue:219]:
```suggestion
            this.$store.dispatch('libraries/fetch', this.currentLibraryId)
```

**Review by advplyr — CHANGES_REQUESTED** [review]
I think this is the right solution so we reload the library data but we can use the `currentLibraryId` instead of passing the library id into the component.

**Comment by barrycarey** [comment]
Right on.  Updated. Didn't catch that the ID was already available.

**Pull request merged**